app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,81 @@
+class RecipesController < ApplicationController
```

Comment:
![image](https://github.com/zbahati/recipeApp/assets/93039359/9e45b4ac-6fcb-446f-af22-2127828e0295)

- [ ]  Great work so far but I cannot see any public recipes even though I have made two recipes public :thinking: Kindly fix this issue to make your project work as expected :+1:

Reply:
For the public recipes when you create the recipe without ingredient even yu put them public it should be visible 
so to be visible you should add at least one ingredient need to the recipe